tests/options/NVM_LAZY_LOAD_EXTRA_COMMANDS

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+source ../common.sh
+
+# Node.js version to install
+node_version=v5.11.0
+
+# Load zsh-nvm and install Node.js in subshell
+(load_zsh_nvm && nvm install "$node_version" && [[ "$(node --version)" == "$node_version" ]]) || die "node wasn't installed"
+
+# Check node isn't available
+[[ "$(node --version)" != "$node_version" ]] || die "node shouldn't be available $(node --version)"
+
+# Set NVM_LAZY_LOAD to true
+export NVM_LAZY_LOAD=true
+export NVM_LAZY_LOAD_EXTRA_COMMANDS=('vim' 'ls')
```

Comment:
`vim` may not exist on all machines where we run tests and `ls` could be called inside other functions inside `zsh-nvm`.

For the test vectors can you choose commonly available yet not that widely used binaries, and things we are unlikely to ever call in `zsh-nvm`.

`whoami` and `hostname` spring to mind.

Reply:
Good point, I'll use `whoami`

Reply:
Can you change `ls` to `hostname` too. Otherwise if we use `ls` somewhere it could cause the test to start failing.